_includes/configuration/image-file-types.md

```diff
@@ -0,0 +1,20 @@
+### `image_file_types`
+
+This option allows you to specify which image file formats will be recognised and modified by [smart_paste](#smart_paste) when pasted.
```

Comment:
... recognised and modified by smart_paste
would "processed" be suitable in this context?

Reply:
`you` is normally a banned word because it's personifying the text or something to that effect. Can this be reworded? eg:
```suggestion
This option configures which image file formats will be recognised and modified by the [smart_paste](#smart_paste) functionality when content is pasted into the editor.
```